tools/topology/topology2/include/components/src_format_s32_convert_from_48k.conf

```diff
@@ -27,7 +27,8 @@
 					]
 					out_bit_depth [ 32 ]
 					out_valid_bit_depth [ 32 ]
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+					# actual buffer size in sof is obs * 2, so here simply set to its period size
+					obs "$[$out_channels * ($out_bit_depth / 8) * (($out_rate / 1000)) * ($period / 1000)]"
```

Comment:
@btian1 But isn't this the same definition we have in the baseline? So if this doesn't require custom obs logic, we can remove this altogether?

Reply:
obs "$[($out_channels * ($[($out_rate + 999)] / 1000)) * ($out_bit_depth / 8)]"

this adding period into count, however, this may change further,  a lot of related changes are happening.
let's wait and see.